### DISCUSSION TITLE
Spectre Importer: mapping destination accounts

### DISCUSSION THREAD

**Discussion by Genygo** [firefly-iii/firefly-iii#4800] [General]
I am running Firefly III version 5.5.11

**Description**
I try to use spectre for importing form my bank. I noticed that my transaction don't have a destination account.
In the import process i checked "map data" but i am only presented with the option to map the spectre categories:

![image](https://user-images.githubusercontent.com/26231125/119671997-84c7cf80-be3a-11eb-9eda-28a22187e0c6.png)

Could this be a limitation from my bank or from saltedge?

Would love to here how other users handle this.

Best regards

**Comment by JC5** [comment]
If the data isn't there then there's not much to map, unfortunately.

**Comment by dmshimself** [comment]
I have the same issue and because there is no payee data (or similar) presented by Saltedge for the transaction for my bank, I don't think this type of import will be useful.  Is that something other people see too?
